File: Basic Coding Problems/Pattern Print/Print simple Pyramid/code.cpp
#include<bits/stdc++.h>
using namespace std;

// Printing Simple Pyramid

int main(){
    int n;
    cin>>n;
    for(int i=0;i<n;i++){
        for(int j=0;j<=i;j++){
            cout<<"* ";
        }
        cout<<endl;
    }
}

File: Basic Coding Problems/Prime or Not/code.go
#include<bits/stdc++.h>
using namespace std;

// Check Prime number

int main(){
    int n,flag=0;
    cin>>n;

    int m = n/2;
    for(int i=2;i<=m;i++){
        if(n%i==0){
            flag =1;
            break;
        }
    }

    if(flag == 0)
        cout<<"Number is Prime";
    else
        cout<<"Number is not Prime";
}
